database/include/mainframe/database/base/row.h

```diff
@@ -18,6 +18,8 @@ namespace mainframe {
 			~Row() = default;
 
 			Row(const std::shared_ptr<std::vector<std::string>>& _columns);
+			const std::shared_ptr<std::vector<std::string>>& getColumns() const;
```

Comment:
`shared_ptr&` doesn't make sense to pass as argument or to return, probably change to normal shared_ptr

Reply:
changed